package.json

```diff
@@ -157,6 +157,16 @@
         "category": "Metals",
         "title": "Restart server"
       },
+      {
+        "command": "metals.super-method-hierarchy",
+        "category": "Metals",
+        "title": "Reveal super method hierachy"
+      },
+      {
+        "command": "metals.go-to-super-method",
+        "category": "Metals",
+        "title": "Go to super method"
+      },
```

Comment:
can you also add them to the list below, in order to enable them only when Metals is enabled?

Reply:
Good idea! We actually forgot about that for some more commands. I added all of the missing ones, just left the `Reveal metals view` ones, since they do work without the server and still do something in that case.

Reply:
Good catch 👍